junit4-ant/src/main/java/com/carrotsearch/ant/tasks/junit4/JUnit4.java

```diff
@@ -919,6 +933,13 @@ public void execute() throws BuildException {
       aggregatedBus.register(o);
     }
 
+    // Create System property if there are any user defined RunListeners
+    if(!runListeners.isEmpty()) {
```

Comment:
Please add space bars after if keyword (here and elsewhere in the patch) to be consistent with the rest of the code.

Reply:
Ok, I will update this.

Reply:
Thanks Steven.